finmodel/src/render.py

```diff
@@ -20,15 +20,18 @@
 # ===================================================================================================================================================
 def render_df(df):
 
+
     # -----------------------------------------------------------------------------------------------------------------------------------------------
     # instantiating various things, configuring
+    warnings.simplefilter(action='ignore', category=FutureWarning) # just doing this for now I promise
     df.reset_index(inplace=True, drop=False)
     df.rename(columns={"Account": "ACCOUNT",
                        "Year": "YEAR",
                        "yoy_growth": "YOY\nGROWTH",
                        "gizmo_mix": "GIZMO\nMIX",
+                       "Source": "SOURCE",
+                       "Sales": "SALES\n(USD)",
                         }, inplace=True)
-
     df.fillna(" ", inplace=True)
```

Comment:
I think this line was throwing a warning for me, didn't like the str with an int64 or float64 array

Reply:
I am suppressing the warning for now on line 26. It's just cause im going in an messing with it purely for rendering purposes. I will find a different fix later.